templates/admin/job_template/index.html.ep

```diff
@@ -105,7 +105,7 @@
         <form action="#" id="editor-form" class="form-horizontal" style="display: none;"
               data-put-url="<%= url_for('apiv1_job_templates_schedules' => (id => $group->id)) %>" data-reference="<%= $group->template %>">
             <div class="form-group row">
-                <label for="editor-template" class="col-sm-4 control-label">
+                <label for="editor-template" class="col-sm-5 control-label">
                     <div id="editor-yaml-guide">
                         <div>Define test suites per arch/medium:</div>
```

Comment:
hm, "define test suites". How about "Define job templates with existing test suites"?

Reply:
Good catch. That would make more sense indeed.